server/sonar-db-migration/src/main/java/org/sonar/server/platform/db/migration/version/v70/CreateTableLiveMeasures.java

```diff
@@ -0,0 +1,119 @@
+import static org.sonar.server.platform.db.migration.def.BlobColumnDef.newBlobColumnDefBuilder;
+import static org.sonar.server.platform.db.migration.def.DecimalColumnDef.newDecimalColumnDefBuilder;
+import static org.sonar.server.platform.db.migration.def.IntegerColumnDef.newIntegerColumnDefBuilder;
+import static org.sonar.server.platform.db.migration.def.VarcharColumnDef.newVarcharColumnDefBuilder;
+
+public class CreateTableLiveMeasures extends DdlChange {
+
+  private static final String TABLE_NAME = "live_measures";
+
+  public CreateTableLiveMeasures(Database db) {
+    super(db);
+  }
+
+  @Override
+  public void execute(Context context) throws SQLException {
+    context.execute(new CreateTableBuilder(getDialect(), TABLE_NAME)
+      .addPkColumn(newVarcharColumnDefBuilder()
+        .setColumnName("uuid")
+        .setIsNullable(false)
+        .setLimit(VarcharColumnDef.UUID_SIZE)
+        .build())
+      .addColumn(newVarcharColumnDefBuilder()
+        .setColumnName("project_uuid")
+        .setIsNullable(false)
+        .setLimit(VarcharColumnDef.UUID_VARCHAR_SIZE)
+        .build())
+      .addColumn(newVarcharColumnDefBuilder()
+        .setColumnName("component_uuid")
+        .setIsNullable(false)
+        .setLimit(VarcharColumnDef.UUID_VARCHAR_SIZE)
+        .build())
+      .addColumn(newIntegerColumnDefBuilder()
+        .setColumnName("metric_id")
+        .setIsNullable(false)
+        .build())
+      .addColumn(newDecimalColumnDefBuilder()
+        .setColumnName("value")
+        .setPrecision(38)
+        .setScale(20)
+        .build())
+      .addColumn(newVarcharColumnDefBuilder()
+        .setColumnName("text_value")
+        .setIsNullable(true)
+        .setLimit(4_000)
+        .build())
+      .addColumn(newDecimalColumnDefBuilder()
+        .setColumnName("variation")
+        .setPrecision(38)
+        .setScale(20)
+        .build())
+      .addColumn(newBlobColumnDefBuilder()
+        .setColumnName("measure_data")
+        .build())
+      .addColumn(newBigIntegerColumnDefBuilder()
+        .setColumnName("created_at")
+        .setIsNullable(false)
+        .build())
+      .addColumn(newBigIntegerColumnDefBuilder()
+        .setColumnName("updated_at")
+        .setIsNullable(false)
+        .build())
+      .build());
+
+    context.execute(new CreateIndexBuilder(getDialect())
+      .addColumn(newVarcharColumnDefBuilder()
+        .setColumnName("project_uuid")
+        .setIsNullable(false)
+        .setLimit(VarcharColumnDef.UUID_VARCHAR_SIZE)
+        .build())
+      .setUnique(false)
+      .setTable(TABLE_NAME)
+      .setName("live_measures_project")
+      .build());
+
+    context.execute(new CreateIndexBuilder(getDialect())
+      .addColumn(newVarcharColumnDefBuilder()
+        .setColumnName("component_uuid")
+        .setIsNullable(false)
+        .setLimit(VarcharColumnDef.UUID_VARCHAR_SIZE)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "UUID_VARCHAR_SIZE"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)